lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -124,19 +144,18 @@ window.runLighthouseForConnection = function(connection, url, options, requested
   lighthouseIsRunning = true;
   updateBadgeUI(url);
 
-  // Run Lighthouse.
-  return Runner.run(connection, runOptions)
-    .then(result => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      filterOutArtifacts(result);
-      return result;
-    })
-    .catch(err => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      throw err;
-    });
+  return Runner.run(connection, runOptions) // Run Lighthouse.
```

Comment:
[We're not consistent](https://github.com/GoogleChrome/lighthouse/search?utf8=%E2%9C%93&q=++.then%28&type=Code), but I'll revert to be consistent with the file. 

If we want to enforce something other than Goog style, there really should be an eslint rule.